index.bs

```diff
@@ -1798,6 +1822,66 @@ Note: Invoking this method from a [=browsing context=] where the [=Web Authentic
 
 </div>
 
+### Client Supported Capabilities (dictionary <dfn dictionary>SupportedCapabilities</dfn>) ### {#dictionary-supportedcapabilities}
+
+<xmp class="idl">
+    dictionary SupportedCapabilities {
+        boolean areU2FDevicesSupported;
+        boolean areCTAPDevicesSupported;
+        boolean areResidentKeysSupported;
+        boolean isUserVerificationSupported;
```

Comment:
As I think was mentioned in the call, `isUserVerificationSupported` seems odd and I think you might want `isClientPINSupported`.

Reply:
I think **isUserVerificationSupported** is more about whether the client (browser) supports **userVerification** option for create() and get() call. Plus, it's better to have isClientPINSupported as well.

Reply:
Do we have other forms of user verification that hte client can do besides clientPin?  If not, I think `isClientPINSupported` is the correct name here.

Reply:
Updated for clientPin as opposed to UV.